webapp/security/alembic/versions/8e9386e7a94b_add_component_column_to_status_table.py

```diff
@@ -0,0 +1,35 @@
+"""add component column to status table
+
+Revision ID: 8e9386e7a94b
+Revises: fc282ca1dc4d
+Create Date: 2020-07-16 15:01:39.739465
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "8e9386e7a94b"
+down_revision = "fc282ca1dc4d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    components = ["main", "universe"]
+    components_str = "'" + "','".join(components) + "'"
+    op.execute(f"CREATE TYPE components AS ENUM ({components_str});")
+    op.add_column(
+        "status",
+        sa.Column(
+            "component",
+            sa.Enum(*components, name="components"),
+            nullable=False,
+            server_default="main",
+        ),
+    )
+
+
+def downgrade():
+    op.drop_column("status", "component")
```

Comment:
I've discovered that you actually need:

``` python
op.execute(f"DROP TYPE components;")
```

As well

Reply:
Added.